docs/source/deployorderer/ordererchecklist.md

```diff
@@ -0,0 +1,320 @@
+
+If you plan to use Prometheus [metrics](#metrics) to monitor your ordering node, you must configure the operations service here.
+
+In the unlikely case where two ordering nodes are running on the same node on your infrastructure, you need to modify the addresses for the second ordering node to use a different port. Otherwise, when you start the second ordering node, it will fail to start, reporting that the addresses are already in use.
+
+```
+# host and port for the operations server
+    ListenAddress: 127.0.0.1:8443
+
+    # TLS configuration for the operations endpoint
+    TLS:
+        # TLS enabled
+        Enabled: false
+
+        # Certificate is the location of the PEM encoded TLS certificate
+        Certificate:
+
+        # PrivateKey points to the location of the PEM-encoded key
+        PrivateKey:
+
+        # Most operations service endpoints require client authentication when TLS
+        # is enabled. ClientAuthRequired requires client certificate authentication
+        # at the TLS layer to access all resources.
+        ClientAuthRequired: false
+
+        # Paths to PEM encoded ca certificates to trust for client authentication
+        ClientRootCAs: []
+```
+
+* **`ListenAddress`**: (required when using the operations service) Specify the address and port of the operations server.
+* **`Enabled`**: (required when using the operations service) Must be `true` if the operations service is being used.
+* **`Certificate`**: (required when using the operations service) Can be the same file as the `General.TLS.Certificate`.
+* **`PrivateKey`**: (required when using the operations service) Can be the same file as the `General.TLS.PrivateKey`.
+* **`ClientAuthRequired`**: (required when using the operations service) Must be set to `true` to enable mutual TLS between the client and the server.
+* **`ClientRootCAs`**: (required when using the operations service) Similar to the client root CA cert file in TLS, it contains a list of client root CA certificates that can be used to verify client certificates. If the client enrolled with the orderer organization CA, then this value is the orderer organization root CA cert.
+
+## Metrics.*
+
+By default this is disabled, but if you want to monitor the metrics for the orderer, you need to use `StatsD` as your metric provider. `StatsD` uses a "push" model, pushing metrics from the orering node to a `StatsD` endpoint. Because of this, it does not require configuration of the operations service itself. See the list of [Available metrics for the orderer](../metrics_reference.html#orderer-metrics).
```

Comment:
These seems like it was copied maybe? But it's very misleading to me.

> if you want to monitor the metrics for the orderer, you need to use `StatsD` as your metric provider

You can monitor metrics using prometheus too?  I would think it should read: "if you want to monitor the metrics for the orderer, you must set this value to either `statsd` or `prometheus`.  Then have a paragraph about each.

Reply:
Seems sufficient to me to just link to the metrics reference doc, but if we want to add a paragraph of information here for each metrics type, I'm open to it. Not being an expert on metrics, I'm not really sure what we should say, though.